README.md

```diff
@@ -23,12 +23,15 @@ Our aim is to provide a single source for the best possible OS X binaries for cg
 * autoconf
 * libtool
 * GNU sed
+* libevent
+
 
 #### BFGMiner
 
 * openssl
 
-Install with Homebrew - e.g. `brew install automake autoconf libtool sed`
+Install with Homebrew - e.g. `brew install automake autoconf libtool gnu-sed libevent`
```

Comment:
See above comment - I'm not seeing a need for `libevent` as a prepreq. I am able to build fine without it and it is downloaded and linked as a dependency for those miners that use it.

Reply:
I removed it, but added wget and git after experimenting on a clean system if that's alright